tests/unit/cms/classes/AssetTest.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+use Cms\Classes\Theme;
+use Cms\Classes\Asset;
+
+class AssetTest extends TestCase
+{
+    public function testLoad()
+    {
+        $theme = Theme::load('test');
+
+        // Forward compatible assertions
+        // @TODO: Use only `assertStringContainsString` after L6 upgrade
+
+        if (method_exists($this, 'assertStringContainsString')) {
```

Comment:
You can just target the L6 branch for this PR, we'll probably merge it in this week.